estate/models/estate_property.py

```diff
@@ -0,0 +1,25 @@
+from odoo import models,fields
+
+class EstateProperty(models.Model):
+    _name = "estate.property"
+    
+    name = fields.Char("Property Name" , required=True)
+    description = fields.Text("Description")
+    postcode = fields.Char("Postcode")
+    date_availability = fields.Date("Date")
+    expected_price = fields.Float("Expected Price")
+    selling_price = fields.Float("Selling Price")
+    bedrooms = fields.Integer("Bedrooms")
+    living_area = fields.Integer("Living Area")
+    facades = fields.Integer("Facades")
+    garage = fields.Boolean("Garage")
+    garden = fields.Boolean("Garden")
+    garden_area = fields.Integer("Garden Area")
+    garden_orientation = fields.Selection( 
+        string="Garden Orientation",
+        selection=[
+        ('north', 'North'), 
+        ('south', 'South'),
+        ('east', 'East'),
+        ('west', 'West')])
```

Comment:
```suggestion
        ('west', 'West')
      ])
```
add new line at the end of very file